srml/staking/src/lib.rs

```diff
@@ -1112,16 +1112,25 @@ impl<T: Trait> Module<T> {
 			}
 
 			// This optimization will most likely be only applied off-chain.
-			let do_equalize = false;
-			if do_equalize {
-				let tolerance = 10 as u128;
-				let iterations = 10 as usize;
-				phragmen::equalize::<T>(
-					&mut assignments_with_stakes,
-					&mut exposures,
-					tolerance,
-					iterations
-				);
+			let do_equalise;
+			if cfg!(feature = "equalise") { do_equalise = true; }
```

Comment:
bad indentation. in general this should be:

```
if CONDITION {
  true
} else {
  false
}
```

however what's wrong with:
```
let do_equalise = cfg!(feature = "equalise");
```

Reply:
hrm, I don't think the variable `do_equalise` makes it more readable

hrm, why not:

```rustc
if cfg!(feature = "equalise") {
   ....
   equalise::<T>(&mut assignments_with_votes, &mut exposures, tolerance, iterations);
}

Reply:
@niklasad1 totally agree I saw this when I re-requested a review earlier today but didn't fix at the time since was reluctant to switch branches and kill all of my build artifacts (perks of temporarily working on substrate with a dual-core-some-five-years-old cpu :p)

Reply:
@Kianenigma

> @niklasad1 totally agree I saw this when I re-requested a review earlier today but didn't fix at the time since was reluctant to switch branches and kill all of my build artifacts (perks of temporarily working on substrate with a dual-core-some-five-years-old cpu :p)

And I thought my quad-core (8T but hyperthreading is disabled by L1TF mitigations) computer was weak.